rotkehlchen/tests/api/test_misc.py

```diff
@@ -220,20 +220,20 @@ def test_manage_nodes(rotkehlchen_api_server: 'APIServer') -> None:
             assert node['blockchain'] == blockchain_key
             break
 
-    # Try to add etherscan as node
+    # Try to add an empty name
     response = requests.put(
         api_url_for(rotkehlchen_api_server, 'rpcnodesresource', blockchain=blockchain_key),
         json={
-            'name': 'etherscan',
```

Comment:
same thing here. you should not be able to add "etherscan" as a node.

Reply:
now you can name it as you want since etherscan is not a reserver name anymore. The only condition now is that the endpoint can't be empty